cosmos/keybase_test.go

```diff
@@ -40,4 +41,47 @@ func TestKeybase(t *testing.T) {
 		require.NoError(t, err)
 		require.False(t, exist)
 	})
+
+	t.Run("Sign", func(t *testing.T) {
+		name2 := "name2"
+		mnemonic2, _ := kb.NewMnemonic()
+		kb.CreateAccount(name2, mnemonic2, "", password, 0, 0)
+		t.Run("Two accounts", func(t *testing.T) {
+			// first
+			hash := sha256.Sum256([]byte(name + password))
+			sig, pub, err := kb.Sign(name, password, []byte{})
+			require.NoError(t, err)
+			require.NotEmpty(t, sig)
+			require.NotEmpty(t, pub)
+			require.NotEmpty(t, kb.privCache[hash])
+			// second
+			hash2 := sha256.Sum256([]byte(name2 + password))
+			sig2, pub2, err := kb.Sign(name2, password, []byte{})
+			require.NoError(t, err)
+			require.NotEmpty(t, sig2)
+			require.NotEmpty(t, pub2)
+			require.NotEmpty(t, kb.privCache[hash2])
+			// not the same
+			require.NotEqual(t, sig, sig2)
+			require.NotEqual(t, pub, pub2)
+			require.NotEqual(t, kb.privCache[hash], kb.privCache[hash2])
+		})
+		t.Run("Many times", func(t *testing.T) {
```

Comment:
What is the purpose for tests "Two accounts" and "Many times", are they checking different thing? can't we just have one of them?